### ISSUE TITLE
Designing Network Design Spaces

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#66]
# TL;DR
Ilija Radosavovic, Raj Prateek Kosaraju, Ross Girshick, Kaiming He, Piotr Dollár
- Facebook AI Research
	

Proposes a network structure optimization method that considers a set of network parameters and finds a good set through experimentation, showing specific evaluation methods and results, although it is not fully automated like NAS, but rather closer to the manual design of a skilled person. The search result RegNet is five times faster and with better accuracy than EfficientNet.

<img width="453" alt="cumulative prob," src="https://user-images.githubusercontent.com/24524018/92213633-bd9bcf00-eece-11ea-80e7-d4ee92654601.png">

# Paper URL
https://arxiv.org/abs/2003.13678

# Submission Dates(yyyy/mm/dd)
2020/05/30

# Authors and institutions

# Methods

# Results

# Comments
